src/azure-cli-core/azure/cli/core/_session.py

```diff
@@ -7,12 +7,19 @@
 import logging
 import os
 import time
-from collections.abc import MutableMapping
-
 from knack.log import get_logger
 
+import collections
+
+
+try:
+    collectionsAbc = collections.abc
+except AttributeError:
+    collectionsAbc = collections
+
+
+class Session(collectionsAbc.MutableMapping):
 
-class Session(MutableMapping):
```

Comment:
I believe this issue has been solved by PR #21877. @jiasli Could you help confirm it?

Reply:
I've rebased my branch onto the dev branch. Thanks for the info!